### PR TITLE
Change max length of street address fields

### PR DESCRIPTION
**Pull request by anders-schneider**
This change doubles the max length of the street address (to accommodate a few Detroit addresses that exceed the 64 character length).

### CODE REVIEW

File: backend/core/src/migration/1625825502154-refactor-unit-types-rent-types-and-priority-types.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export class refactorUnitTypesRentTypesAndPriorityTypes1625825502154 implements
       `ALTER TABLE "units" ADD CONSTRAINT "FK_1e193f5ffdda908517e47d4e021" FOREIGN KEY ("unit_type_id") REFERENCES "unit_types"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`
     )
 
-    const unitTypeSeeds = ["studio", "oneBdrm", "twoBdrm", "threeBdrm", "fourBdrm"]
+    const unitTypeSeeds = ["studio", "oneBdrm", "twoBdrm", "threeBdrm", "fourBdrm", "fiveBdrm"]
```
Comment (willrlin) [inline backend/core/src/migration/1625825502154-refactor-unit-types-rent-types-and-priority-types.ts:15]: If we're planning to submit this upstream you'd probably need to generate a new migration, no?

File: backend/core/src/shared/entities/address.entity.ts [hunk 2]
```diff
@@ -55,14 +55,14 @@ export class Address {
   @Expose()
   @IsOptional({ groups: [ValidationsGroupsEnum.partners] })
   @IsString({ groups: [ValidationsGroupsEnum.default] })
-  @MaxLength(64, { groups: [ValidationsGroupsEnum.default] })
+  @MaxLength(128, { groups: [ValidationsGroupsEnum.default] })
```
Comment (willrlin) [inline backend/core/src/shared/entities/address.entity.ts:58]: Does this also need a migration?

**Review by willrlin — COMMENTED** [review]
How common is the 5 BR? I'm hesitant about this change since I would like Bloom to also accept it, but I'm wondering what would need to happen in order to make that happen.

File: backend/core/src/shared/entities/address.entity.ts (hunk 2 above)
Comment (anders-schneider) [reply backend/core/src/shared/entities/address.entity.ts:58]: No, it looks like this is just a typeorm-level validation, not a DB-level validation (`yarn db:migration:generate` doesn't generate anything for this)

File: backend/core/src/migration/1625825502154-refactor-unit-types-rent-types-and-priority-types.ts (hunk 1 above)
Comment (anders-schneider) [reply backend/core/src/migration/1625825502154-refactor-unit-types-rent-types-and-priority-types.ts:15]: Yeah I think so. (My mental model for how it works when there's a new migration: only the new migrations are applied to the deployed DB, there's never a `yarn db:reseed`-like step that re-applies all migrations for the deployed DB.)

**Comment by anders-schneider** [comment]
Only 4 Detroit listings have non-zero Num_5BR. So I suppose we could just add those counts to the "fourBdrm" unit type count and understand "fourBdrm" to mean "four or more bedrooms".

My initial feeling: it doesn't seem important to push this change to Exygy since it seems like a simple difference between their use case and ours. But I'm also happy to just drop "fiveBdrm" altogether since it's so few listings.

Hoping you have a strong feeling one way or another :)

**Comment by willrlin** [comment]
Yeah, I'm inclined to drop the 5BR altogether since it's so few listings and the fact that they already defined 4 as a string to have "4+". We can also check with @kar-r-green if it's a product requirement. Let's leave it out for now, but maybe save that portion of this commit on a branch somewhere.

File: backend/core/src/migration/1625825502154-refactor-unit-types-rent-types-and-priority-types.ts (hunk 1 above)
Comment (willrlin) [reply backend/core/src/migration/1625825502154-refactor-unit-types-rent-types-and-priority-types.ts:15]: The bonus of dropping it is we won't have to deal with this migration!

Comment (anders-schneider) [reply backend/core/src/migration/1625825502154-refactor-unit-types-rent-types-and-priority-types.ts]: Sounds good :)

**Comment by anders-schneider** [comment]
SGTM - reverted the 5br changes and updated the PR title/description!

**Review by willrlin — APPROVED** [review]

**Pull request merged**